source/_integrations/hisense_aehw4a1.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: "Hisense AEH-W4A1"
+description: "Instructions to setup the Hisense AEH W4A1 wifi module for ACs."
+logo: hisense.png
+ha_release: 0.102
+ha_category:
+  - Climate
+ha_iot_class: Local Pull
+---
+
+The Hisense AEH-W4A1 is wifi module used to give wifi connectivity to some Hisense ACs and rebranded models (Smart Cool, Beko and others).
+
+Later AC models use other wifi modules (like AEH-W4B1 and AEH-W4E1) that are not compatible or otherwise tested  with this integration.
```

Comment:
```suggestion
Later AC models use other WiFi modules (like AEH-W4B1 and AEH-W4E1) that are not compatible or otherwise tested with this integration.
```